csun-metalab/nomi
# NAM-553 -- UX: Insufficient contrast for bottom navigation inactive buttons

# Description
Text under buttons that are in inactive state (not active in navigation) do not have enough contrast against the background.
  
# Testing
1. Login as `steve`.
2. Ensure the bottom `nav-bar` is rated for `WCAG AA Contrast` standard and looks somewhat pleasing. 
` Ignore the disabled button color ` , reason being listed down below. 

***` "WCAG states that Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement." `***
